modules/mobile/app/helpers/mobile/pagination_helper.rb

```diff
@@ -69,6 +69,7 @@ def url_without_page_number
           query_strings << "#{camelized}=#{validated_params[key]}"
         end
 
+        query_strings << 'included[]=pending' if validated_params[:included]&.include?('pending')
```

Comment:
Maybe we could subclass the pagination helper for appointments and override this method? Claims, payments, etc. don't know about 'pending' things :)

Reply:
Or `link_url` could be simpler to override, just tack on the include if necessary.

Reply:
I'll remove this part of the PR and deal with it in the next PRs.

Reply:
Removed.